src/app/config/projectProperties.json

```diff
@@ -0,0 +1,68 @@
+{
+    "projectTypes": {
+        "angular": {
+            "name": "angular",
+            "displayname": "Office Add-in project using Angular framework",
+            "repolocation": "",
```

Comment:
"repository" would be simpler than "repolocation".

Reply:
You'll really need a repository location per ts/js language. I think it would be better to have a languages property with a sub-object with "javascript" and "typescript" which for each has "repository".

Reply:
Something like this:
```
propertyTypes: {
     "excel-functions": {
          "displayName": "Excel Custom Functions Add-in",
          "templates: {
             "typescript": {
                    "repository: "https://www.github.com/OfficeDev/office-addin-templates-excel-functions"
             },
             "javascript": {
                    "repository: "https://www.github.com/OfficeDev/office-addin-templates-excel-functions"
             }
    }
```